src/testRunner/unittests/tsc/helpers.ts

```diff
@@ -10,7 +10,15 @@ namespace ts {
         subScenario: "no-change-run",
         modifyFs: noop
     };
+    export const noChangeWithExportsDescripencyRun: TestTscEdit = {
+        ...noChangeRun,
+        descripencyExplaination: () => [
+            "Incremental build does not change and has .ts as signature so exports has all imported modules/referenced files",

```

Comment:
Does "Incremental build does not change" mean "Incremental build inputs did not change"?